src/ngraph/op/batch_norm.hpp

```diff
@@ -36,6 +36,15 @@ namespace ngraph
                       std::shared_ptr<Node> beta,
                       std::shared_ptr<Node> input);
 
+            // BatchNorm Training
+            BatchNorm(bool training,
```

Comment:
IMO it's a bit confusing to have a comment indicating that this is a "training" version of the c'tor, but then to also parameter whose name suggests that "training" is optional.

Reply:
I agree. I think it'd be better to extend the existing constructor with a boolean parameter in the last position, whose default value is `false`. The behavior of this constructor is identical to the existing five-argument inference constructor.